src/deepsparse/yolact/schemas.py

```diff
@@ -68,8 +69,31 @@ class YOLACTInputSchema(ComputerVisionSchema, Splittable):
         "`postprocess` step (optional)",
     )
 
+    @classmethod
+    def from_files(cls, files: List[str], **kwargs) -> "YOLACTInputSchema":
+        """
+        :param files: list of file paths to create YOLOInput from
+        :param kwargs: extra keyword args to pass to YOLOInput constructor
+        :return: YOLOInput constructed from files
+        """
+        if "images" in kwargs:
+            raise ValueError(
+                f"argument 'images' cannot be specified in {cls.__name__} when "
+                "constructing from file(s)"
+            )
+        files_numpy = [numpy.array(Image.open(file)) for file in files]
+        input_schema = cls(images=files_numpy, **kwargs)
+        # segmentation masks for now are too big to send them
+        # over the https server. They are excluded from the
+        # response payload
+        input_schema.Config.remove_masks_from_output = True
```

Comment:
Will this force the input_schema to never return a mask if `from_files` is used? What if a user tries to create inputs from a bunch of local files? Will the masks not be returned then?

Reply:
Yes, you are right. We remove masks from the payload whenever we use the server for yolact inference.
We could think about disabling it only when we receive remote files, good point. I guess this could probably be inferred from the properties of the starlette file.